astropy/cosmology/realizations.py

```diff
@@ -0,0 +1,112 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import sys
+import warnings
+
+from astropy import units as u
+from astropy.utils.exceptions import AstropyDeprecationWarning
+from astropy.utils.state import ScienceState
+
+from . import parameters
+from .core import Cosmology, FlatLambdaCDM, LambdaCDM
+
+__all__ = ["default_cosmology"] + parameters.available
+
+__doctest_requires__ = {"*": ["scipy"]}
+
+# Pre-defined cosmologies. This loops over the parameter sets in the
+# parameters module and creates a LambdaCDM or FlatLambdaCDM instance
+# with the same name as the parameter set in the current module's namespace.
+# Note this assumes all the cosmologies in parameters are LambdaCDM,
+# which is true at least as of this writing.
+
+for key in parameters.available:
+    par = getattr(parameters, key)
+    if par["flat"]:
+        cosmo = FlatLambdaCDM(
+            par["H0"],
+            par["Om0"],
+            Tcmb0=par["Tcmb0"],
+            Neff=par["Neff"],
+            m_nu=u.Quantity(par["m_nu"], u.eV),
+            name=key,
+            Ob0=par["Ob0"],
+        )
+        docstr = "{} instance of FlatLambdaCDM cosmology\n\n(from {})"
+        cosmo.__doc__ = docstr.format(key, par["reference"])
+    else:
+        warnings.warn("Please open a PR for your added cosmology realization.")
+        # cosmo = LambdaCDM(
```

Comment:
Could you add a comment here explaining that for a non-flat LCDM model, it would look like this?

Reply:
Thanks. Good suggestion. Done!
```
# For a non-flat LCDM realization, the following is the code necessary
# to create the realization, given the parameters. We comment this out
# as there are not currently any such built-in realizations.
```